tests/installation/test_executor.py

```diff
@@ -111,7 +111,7 @@ def test_execute_executes_a_batch_of_operations(
     env: MockEnv,
 ):
     pip_editable_install = mocker.patch(
-        "poetry.installation.executor.pip_editable_install", unsafe=not PY36
+        "poetry.installation.executor.pip_editable_install", unsafe=not PY37
```

Comment:
this should now always evaluate to `False`, but since that's the [default](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock) i think we can remove it:
```suggestion
        "poetry.installation.executor.pip_editable_install"
```